lib/cli/writer.js

```diff
@@ -67,6 +69,15 @@ exports.writeImage = (imagePath, drive, options, onProgress) => {
     return fs.openAsync(drive.raw, 'rs+');
   }).then((driveFileDescriptor) => {
     return imageStream.getFromFilePath(imagePath).then((image) => {
+      if (!image.size.final.estimation && !constraints.isDriveLargeEnough(drive, {
+        size: image.size.final.value
+      })) {
+        throw errors.createUserError(
+          'The image you selected is too big for this drive',
+          'Plug a bigger drive and try again'
```

Comment:
Should this be "Plug in [...]"? Also, do we want to refer to "drives" as "devices", as most of them these days aren't technically drives?